data_steward/validation/achilles.py

```diff
@@ -29,10 +31,7 @@ def load_analyses(hpo_id):
     """
     project_id = app_identity.get_application_id()
     dataset_id = bq_utils.get_dataset_id()
-    if hpo_id is None:
-        table_prefix = ""
-    else:
-        table_prefix = f'{hpo_id}_'
+    table_prefix = resources.get_table_id(table_name="", hpo_id=hpo_id)
```

Comment:
I think here we can consolidate the line 34 and 35 to 
`table_name = resources.get_table_id(table_name=ACHILLES_ANALYSIS, hpo_id=hpo_id)`

Reply:
This works too.